modules/Bio/EnsEMBL/Compara/PipeConfig/EPOwith2x_conf.pm

```diff
@@ -215,8 +215,10 @@ sub core_pipeline_analyses {
         },
 
         {   -logic_name => 'check_for_lastz',
-            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::Dummy',
-            -flow_into  => WHEN( '#lastz_complete#' => { 'create_default_pairwise_mlss' => { 'mlss_id' => '#low_epo_mlss_id#' }}),
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::CheckSwitch',
+            -parameters => { 'switch_name' => 'lastz_complete' },
+            -flow_into  => { 1 => { 'create_default_pairwise_mlss' => { 'mlss_id' => '#low_epo_mlss_id#' }}},
```

Comment:
```suggestion
            -parameters => {
                'switch_name' => 'lastz_complete',
            },
            -flow_into  => {
                1 => { 'create_default_pairwise_mlss' => { 'mlss_id' => '#low_epo_mlss_id#' } },
            },
```
(just to follow our usual indentation, plus less lines edited if we need to add more to it in the future)